new/LIT6907MastabqMar.xml

```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6907MastabqMar" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መስተብቍዕ፡ ዘማርያም፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Mastabqʷǝʿ za-Māryām</title>
+                <title xml:id="t2">Mastabqʷǝʿ za-Māryām II</title>
+                <title xml:lang="gez" xml:id="t3">ወካዕበ፡ ናስተበቍዓ፡ ለንግሥተ፡ ኵልነ፡</title>
+                <title xml:lang="gez" corresp="#t3">wa-kāʿba nāstabaqqʷǝʿā la-nǝgǝśta kʷǝllǝna</title>
+                <title xml:lang="gez" xml:id="t4">ዓዲ፡ መስተብቍዕ፡ ዘማርያም፡</title>
+                <title xml:lang="gez" corresp="#t4" type="normalized">ʿĀdi Mastabqʷǝʿ za-Māryām</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="ESakm009"></witness>
+                    <witness corresp="BLorient545"></witness>
+                    <witness corresp="BNFet13"></witness>
+                    <witness corresp="EMML1434"></witness>
+                    <witness corresp="EMML4190"></witness>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>It is one of the prayers that are part of the fixed collection <ref target="#LIT4713Liton"/>, but also circulate independently 
```

Comment:
`<ref type="work" corresp="LIT4713Liton"/>` here and below

Reply:
Yes, I changed it in all records.